shaders/Membrane.shader

```diff
@@ -25,16 +29,26 @@ void vertex(){
 }
 
 void fragment(){
-    vec4 normal = texture(albedoTexture, UV);
+    vec4 albedo = texture(albedoTexture, UV);
+    
+    vec4 normalmap = texture(normalTexture, UV);
+    vec3 normals = vec3 (normalmap.r * 2.0 - 1.0, normalmap.g * 2.0 - 1.0, 
+    normalmap.b * 2.0 - 1.0);
+
+    float y = normals.y;
+    normals.y = -normals.z;
+    normals.z = y;
```

Comment:
Why is this flipping done here, instead of like having the normal image file already have correct data?

Reply:
that's how blender saves the bakes. I could probably try to mess with blender to make it bake correctly but, this was faster as i already know how to do this.

Reply:
I guess we just have to live with this, then. As long as it is like the default option. It would be very confusing in the future if new normal maps are made and suddenly they end up looking wrong because of this swapping.